.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
We currently do not have this ESLint config in our application. Where are you getting this from?

Reply:
It's a snippet I got from the houndCI documentation to configure hound to use ESlint as the s tyle guide for our application when ESlint gets configured for our application

Reply:
Yes, but it says to point to a configuration file. When you read the documentation, you should strive towards understanding exactly how the product works, and not just use the snippet.

Reply:
okay, got it.